src/NLog/LayoutRenderers/DirectorySeparatorLayoutRenderer.cs

```diff
@@ -47,11 +47,20 @@ namespace NLog.LayoutRenderers
     [AppDomainFixedOutput]
     public class DirectorySeparatorLayoutRenderer : LayoutRenderer, IRawValue
     {
-        private static readonly char SeparatorChar;
+        private static char? _separatorChar;
```

Comment:
Think this is overly complex, when looking at the implementation for [Path.DirectorySeparatorChar](https://referencesource.microsoft.com/#mscorlib/system/io/path.cs,56)

Think non-static `private char _separatorChar = Path.DirectorySeparatorChar` will work just fine.

Reply:
Fixed!